package.json

```diff
@@ -39,10 +39,11 @@
   "dependencies": {
     "babel-code-frame": "^6.22.0",
     "builtin-modules": "^1.1.1",
-    "chalk": "^2.1.0",
+    "chalk": "~2.1.0",
```

Comment:
I'll look into that and open a separate PR. The change shouldn't be necesary if you use `yarn` instead of `npm`